shoop/admin/modules/products/views/package/forms.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+import six
+from django import forms
+from django.contrib import messages
+from django.db.transaction import atomic
+from django.forms.formsets import BaseFormSet, DELETION_FIELD_NAME
+from django.utils.translation import ugettext_lazy as _
+
+from shoop.admin.forms.widgets import ProductChoiceWidget
+from shoop.core.excs import ImpossibleProductModeException, Problem
+from shoop.core.models import Product, ProductPackageLink
+
+
+def clear_existing_package(parent_product):
+    """
+    Utility function for clearing existing package.
+    """
+    children = parent_product.get_package_child_to_quantity_map().keys()
+    ProductPackageLink.objects.filter(parent=parent_product).delete()
+    parent_product.verify_mode()
+    parent_product.save()
+    for child in children:
+        child.verify_mode()
+        child.save()
+    return True
+
+
+class PackageChildForm(forms.Form):
+    child = forms.ModelChoiceField(
+        queryset=Product.objects.all(),
+        widget=ProductChoiceWidget(),
+        label="",
+        required=True,
+    )
+    quantity = forms.IntegerField(
```

Comment:
I don't think that will work since that is a model field and this is a form field